Header header

uint16 product_id
uint16 serial_number
uint16 version

bool output_GPS_data
bool output_imu_data
bool output_sensor_data
bool need_proofreading
bool using_imu
bool using_geomagnetic
bool using_temp
bool using_pressure
bool using_GPS

int16 data_type
int16 frequency

string imu
string ahrs
string ins
string GPS

string calc_mode
string GPS_status
string acc_status
string malfunction
string mounting_correction
